src/vision/status_quo/niklaus_lost_his_context.md

```diff
@@ -0,0 +1,99 @@
+# 😱 Status quo stories: Niklaus lost his context!
+
+## 🚧 Warning: Draft status 🚧
+
+This is a draft "status quo" story submitted as part of the brainstorming period. It is derived from real-life experiences of actual Rust users and is meant to reflect some of the challenges that Async Rust programmers face today. 
+
+If you would like to expand on this story, or adjust the answers to the FAQ, feel free to open a PR making edits (but keep in mind that, as they reflect peoples' experiences, status quo stories [cannot be wrong], only inaccurate). Alternatively, you may wish to [add your own status quo story][htvsq]!
+
+## The story
+
+Niklaus heard about a project to reimplement a deprecated browser plugin using Rust and WASM. This old technology had the ability to load resources over HTTP; so it makes sense to try and implement that functionality using the Fetch API. He looks up the documentation of `web_sys` and realizes that he needs to take a thing called a `Future` out of the call to `fetch`, do some things to it in Rust, and then send the future off to `spawn_local` as such:
```

Comment:
Does Niklaus understand what `fetch` and `spawn_local` are?

Reply:
I imagine Niklaus knows about them in the sense that...

1. `fetch` lets him call out to JavaScript's `window.fetch`
2. `spawn_local` takes a Rust `Future` and somehow makes it "go"

He presumably does not (yet) understand that `spawn_local` needs to own the futures it runs, or why the lifetimes work out this way. Or why the application he's PRing to is architectured in such a way that makes async difficult.

Reading your other comment I think Alan might have been the better choice and I may rewrite this SQ accordingly.